scripts/release.py

```diff
@@ -228,6 +227,44 @@ def ensure_clean_git() -> None:
         sys.exit(1)
 
 
+def validate_version(version: Text) -> bool:
+    """
+    Ensure that the version follows semver
+    and that the alpha follows the format `alpha1`, `alpha2`, etc...
+    """
+    if not semantic_version.validate(version):
+        return False
+
+    version_object = Version.coerce(version)
+    return not version_object.prerelease or (
+        len(version_object.prerelease) == 1
+        and validate_alpha_version_part(version_object.prerelease[0])
+    )
+
+
+def validate_alpha_version_part(alpha_version_part: Text) -> bool:
```

Comment:
optional: to simplifiy the above return, I'd rename this to `is_alpha_version` and as a parameter I'd use `version: Version`, moving the 
```
(
        len(version_object.prerelease) == 1
        and validate_alpha_version_part(version_object.prerelease[0])
    )
```
code part into this property.